libraries/AP_Hygrometer/AP_Hygrometer_Backend.cpp

```diff
@@ -0,0 +1,27 @@
+/*
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "AP_Hygrometer.h"
+
+#if AP_HYGROMETER_ENABLED
+
+#include "AP_Hygrometer_Backend.h"
+
+AP_Hygrometer_Backend::AP_Hygrometer_Backend(AP_Hygrometer::Param &_params) :
+    params(_params)
+{
+}
```

Comment:
Could just put this in the header and do without the cpp file for now?

Reply:
> Could just put this in the header and do without the cpp file for now?

Yes.